run-docker-tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+PATH=/usr/local/bin:$PATH
```

Comment:
There's no reason/need to override the path

Reply:
The idea is to run the python version that the image ships, which is located in `/usr/local/bin`, otherwise, it would run the tests with an older version (`/usr/bin`).

I can change that to an absolute path, if preferred.

Reply:
It seems `/usr/local/bin` [is there by default](https://docs.docker.com/engine/reference/run/#env-environment-variables)

Reply:
In any case, this script has no knowledge of the image being run, so any PATH manipulations should be done on the `docker run` command in the doc, that does know.